app/src/main/java/org/mozilla/tv/firefox/pocket/PocketViewModel.kt

```diff
@@ -57,6 +57,8 @@ class PocketViewModel(
                 is PocketVideoRepo.FeedState.Inactive -> State.NotDisplayed
             }
         }
+        .replay(1)
+        .autoConnect(0)
```

Comment:
Do we have to specify this every time we don't want observable to autoconnect?

Reply:
I'm planning on giving a presentation soon on some of this, but here's the abridged version.

There are hot and cold observables.  Most `Observables` are cold, and they will start producing emissions whenever you subscribe to them.  Hot `Observables` are ones where the `Observer` can't control the source (e.g., with an `Observable<TouchEvent>`, you can't ask the user to not touch the screen until you've subscribed).  `autoConnect(n)` means "take this hot `Observable`, and tell it that it can start emitting values as soon as n `Observers` have subscribed to it".  So in this case, "start emitting results immediately, even if no one is listening".  `replay(n)` means it will cache the n most recently values it saw.  The end result is that we will always immediately have the most recently seen value cached, and it'll be emitted as soon as someone subscribes.

Reply:
To follow up, most `Observables` you work with will be cold, so you won't need `autoconnect` or any similar operators.  Whenever working with a hot `Observable`, you will need to tell it when to start emitting data.